packages/story-editor/src/components/header/buttons/publish.js

```diff
@@ -105,13 +109,21 @@ function PublishButton({ forceIsSaving }) {
   }, [refreshPostEditURL, saveStory, hasFutureDate, title, canPublish]);
 
   const handlePublish = useCallback(() => {
-    if (shouldReviewDialogBeSeen && canPublish) {
+    if (
+      (shouldReviewDialogBeSeen && canPublish) ||
+      (isUpdatedPublishModalEnabled && canPublish)
+    ) {
```

Comment:
💭  could be simplified to
```suggestion
    if (
       canPublish &&
      (shouldReviewDialogBeSeen ||
       isUpdatedPublishModalEnabled)
    ) {
```

Reply:
Sure!